### ISSUE TITLE
Provide Ubuntu Repository for Dash Core

### ISSUE THREAD

**Issue by Nutomic** [dashpay/dash#1475]
It seems like there is currently no way for Dash on Ubuntu to have automatic updates. It would be great if you could create a repository, so that the project can be easily updated. Ideally, the repository would also install a systemd configuration to run Dash in the background.

https://help.ubuntu.com/community/Repositories

For example, here are some repositories for Bitcoin:
https://launchpad.net/~bitcoin/+archive/ubuntu/bitcoin
https://launchpad.net/~bitcoin-unlimited/+archive/ubuntu/bu-ppa
https://launchpad.net/~bitcoinclassic/+archive/ubuntu/bitcoinclassic

**Comment by Nutomic** [comment]
I just realized that there's already a ppa [here](https://launchpad.net/~dash.org/+archive/ubuntu/dash). Maybe that could be added to the download links on dash.org?

**Comment by nmarley** [comment]
I don't think that PPA is actually used, as the last update that I see there is from almost a year ago, and the most recent Dash version was released in April 2017.

**Comment by schinzelh** [comment]
I'll update the ppa till end of this week - I am maintainer of it, but it slipped through somehow.

**Comment by Nutomic** [comment]
Are you updating the ppa manually? It should be possible to set it up so that new releases are pushed there automatically.

**Comment by PanderMusubi** [comment]
Yes, please automatically update the PPA to prevent users installing old version or assuming they get automated updates while they are not.

**Comment by PanderMusubi** [comment]
And please build for zesty 17.04 and artful 17.10.

**Comment by nmarley** [comment]
Hey @schinzelh , do we still maintain this PPA? If so, should we engage our devops team in getting a plan to automate this?

**Comment by schinzelh** [comment]
Hi,

yes are are still maintaining it - and i agree devops should take this over :)
